themes/default/content/_index.md

```diff
@@ -39,96 +39,457 @@ overview:
-
-    - title: Automate worldwide delivery
-      description: Scale up your delivery with [advanced automation](/automation/) as you grow.
+    Pulumi gives you a faster outer dev loop by making CI/CD for your IaC seamless and the default experience. Pulumi has integrations with all the popular CI/CD platforms and testing frameworks, so you can validate every change with testing and built-in policies. You can also build Pulumi Packages to create best-practice abstractions available in all languages.
 
 manage:
-  title: Manage
+  title: Deliver ideas with ease
   description: |
-    Manage cloud applications and infrastructure with a shared platform that helps teams adopt [Cloud Engineering](/cloud-engineering/) through collaboration, visibility, and policies and controls.
-  items:
-    - title: Secure by default
-      description: Automatic encryption for secrets and state.
-
-    - title: Empower with guardrails
-      description: Identity and Policy as Code ensure continuous compliance.
-
-    - title: See who changed what and when
-      description: Full change visibility and auditing across your entire organization.
-
-benefits:
-  title: The benefits of using Pulumi
-  items:
-    - title: Tame modern cloud complexity
-      icon: code-window
-      icon_color: salmon
-      description: |
-        Build with over 60 cloud providers using a consistent, cloud engineering approach that
-        works for developers and infrastructure teams and eliminates accidental complexity.
-
-    - title: Bring the cloud closer to application development
-      icon: download-from-cloud
-      icon_color: violet
-      description: |
-        Leverage a rich, programmable cloud interface and [reusable packages](/product/packages/) that abstract
-        away complexity, bringing cloud concepts closer to application architectures.
-
-    - title: Use engineering practices with infrastructure
-      icon: exchange
-      icon_color: blue
-      description: |
-        Do more with less. Combine Infrastructure as Code automation with tried-and-true software engineering
-        practices &mdash; including modularity, testing, and CI/CD.
-    - title: Foster collaboration and innovate faster
-      icon: lightning
-      icon_color: yellow
-      description: |
-        Unite developers, infrastructure teams, and security engineers around a common platform
-        so that everyone can speak the same language and ship quickly and reliably.
+    Pulumi gives you a faster dev loop across the entire organization by guaranteeing the infrastructure software supply chain. Standard software packaging allows sharing and reuse of code across the organization along with org-wide policy enforcements. Pulumi provides the industry’s only automation workflow capability that allows software engineering to be applied to solve and manage cloud infrastructure at scale.
+
+automation_api_examples:
+  - name: Provision resources over HTTP (Node.js/Express)
```

Comment:
I'd definitely expand this node by default, as it's really what tells the story, and add a few comments to the code to call out the most relevant parts (and delete whatever we can to keep it short): 

* Import Pulumi as a Node.js library
* Have the program actually deploy something super-simple
* Call out the create and update steps especially
* Drop the try/catch, 'cause who cares, this is just an illustration